app/models/message.rb

```diff
@@ -0,0 +1,11 @@
+class Message < ActiveRecord::Base
+
+  belongs_to :sender, class_name: 'User'
+  belongs_to :recipient, class_name: 'User'
+
+  validates_presence_of :body, :sender_id, :recipient_id
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.